src/components/sortableGridList/types.ts

```diff
@@ -16,6 +16,11 @@ export interface SortableGridListProps<T = any> extends GridListBaseProps, Scrol
    * Temporary migration flag for enabling flex on the container of the list (like it should be by default)
    */
   flexMigration?: boolean;
+  /**
+   * Wether to reorder the items by index instead of by replacing locations.
+   * Items will move to the new index by pushing other items ahead or aback instead of swapping places with the item at the new index.
```

Comment:
Why does this has added description, don't we want it in the API?

Reply:
I felt like the docs should be more concise. Do you think we should add it to the docs?